modules/webbrowser/CMakeLists.txt

```diff
@@ -68,8 +68,7 @@ set(BUILD_SHARED_LIBS ${BUILD_SHARED_LIBS_TMP})
 
 # Determine the target output directory.
 # Inviwo: use module target directory as output dir$<TARGET_FILE_DIR:inviwo-module-webbrowser>
-#SET_CEF_TARGET_OUT_DIR()
-set(CEF_TARGET_OUT_DIR $<TARGET_FILE_DIR:inviwo-module-webbrowser>)
+set(CEF_TARGET_OUT_DIR ${EXECUTABLE_OUTPUT_PATH}/$<CONFIG>)
```

Comment:
on a non multi config build like make or ninja there will not be a CONFIG dir

Reply:
:+1: didn't think about that, fixed now